server/routes/induction/common/wantToAddQualificationsController.ts

```diff
@@ -43,6 +44,34 @@ export default abstract class WantToAddQualificationsController extends Inductio
     )
     return res.render('pages/induction/prePrisonEducation/wantToAddQualifications', { ...view.renderArgs })
   }
+
+  protected formSubmittedFromCheckYourAnswersWithNoChangeMade = (
```

Comment:
These methods have been moved from the `WantToAddQualificationsUpdateController` subclass up into the superclass here, so that they can be used by the `WantToAddQualificationsCreateController` subclass

(basically the desired functional existed in the Update controller, but not the Create controller; so moving it here allows both to use it)